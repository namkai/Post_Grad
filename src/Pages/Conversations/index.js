import React from 'react';
import { connect } from 'react-redux';
import { selectedConversation } from './Actions/index';
import ConversationList from './Components/ConversationList';
import Messages from './Components/Messages';

const Conversations = ({ conversations, messages }) => {
  console.log('i\'m the messages! in CONVO', messages);
  return (
    <div className="modal fade" id="msgModal" tabIndex="-1" role="dialog" aria-labelledby="msgModal" aria-hidden="true">
      <div className="modal-dialog">
        <div className="modal-content">
          <div className="modal-header">
            <h5 className="modal-title">Conversations</h5>
            <button type="button" className="close" data-dismiss="modal" aria-hidden="true">&times;</button>
          </div>

          <div className="modal-body p-0 js-modalBody">
            <div className="modal-body-scroller">
              <div className="media-list media-list-users list-group js-msgGroup">
                <ConversationList conversations={conversations} selectedConversation={selectedConversation} />
              </div>
              <Messages messages={messages} />
            </div>
          </div>
        </div>
      </div>
    </div>
  );
};

export default connect(
	({ conversations, user, messages }) => ({ conversations, user, messages }),
	{ selectedConversation })(Conversations);
