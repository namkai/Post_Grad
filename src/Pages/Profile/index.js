import React from 'react';
import ProfileHeader from './Components/ProfileHeader';

const Profile = props => (
	<div>
		<ProfileHeader />
	</div>
);

export default Profile;
