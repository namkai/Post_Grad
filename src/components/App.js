import firebase from 'firebase';
import 'firebase/auth';
import React, { Component } from 'react';
import Navbar from './Navbar';

class App extends Component {
	componentWillMount() {
		firebase.initializeApp({
			apiKey: 'AIzaSyCs6JKsjx0pMAAXLU4fFHNlkTdnqfHpjdw',
			authDomain: 'portfolio-d6396.firebaseapp.com',
			databaseURL: 'https://portfolio-d6396.firebaseio.com',
			projectId: 'portfolio-d6396',
			storageBucket: 'portfolio-d6396.appspot.com',
			messagingSenderId: '892183671230',
		});
		const token = window.localStorage.getItem("PostGrad");
	}

	render() {
		return (
			<div className="with-top-navbar">
				<Navbar />
			</div>
		);
	}
}

export default App;
