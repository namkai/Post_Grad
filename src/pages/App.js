import firebase from 'firebase';
import 'firebase/auth';
import React, { Component } from 'react';
import { connect } from 'react-redux';
import { authenticate } from '../entities/authentication/actions/index';
import Navbar from '../shared/components/Navbar';
import Messages from './Conversations/index';

export class App extends Component {
	componentWillMount() {
		firebase.initializeApp({
			apiKey: 'AIzaSyCs6JKsjx0pMAAXLU4fFHNlkTdnqfHpjdw',
			authDomain: 'portfolio-d6396.firebaseapp.com',
			databaseURL: 'https://portfolio-d6396.firebaseio.com',
			projectId: 'portfolio-d6396',
			storageBucket: 'portfolio-d6396.appspot.com',
			messagingSenderId: '892183671230',
		});
		firebase.auth().onAuthStateChanged((user) => {
			if (user) {
				console.log(user);
//        this.props.authenticate();
			}
		});
	}

	render() {
		return (
			<div className="with-top-navbar">
				<Navbar />
				<Messages />
			</div>
		);
	}
}

export default connect(null, authenticate)(App);
