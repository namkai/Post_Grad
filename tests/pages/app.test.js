import React from 'react';
import { App } from '../../src/Pages/App';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('App', () => {
	it('Renders a app', () => {
		const app = shallow(<App />);
		expect(toJson(app)).toMatchSnapshot();
	});
});