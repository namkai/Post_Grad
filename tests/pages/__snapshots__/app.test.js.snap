exports[`App Renders a app 1`] = `
<div
  className="with-top-navbar">
  <Connect(Navbar) />
  <Connect(Messages) />
</div>
`;
